### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -8,6 +8,14 @@ This document includes a list of open source components used in Mattermost Web A
 
 --------
 
+## @types/highlight
+
+---
+
+## @typescript-eslint/parser
```
Comment (amyblais) [inline NOTICE.txt]: Was added via https://github.com/mattermost/mattermost-webapp/commit/fef8d8d2778e383b7750d5247a513e3ea03cd0a0#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

File: NOTICE.txt [hunk 2]
```diff
@@ -8,6 +8,14 @@ This document includes a list of open source components used in Mattermost Web A
 
 --------
 
+## @types/highlight
```
Comment (amyblais) [inline NOTICE.txt]: Was added via https://github.com/mattermost/mattermost-webapp/commit/97fd5cbf82ba9b583d10ec7380d8c31048d3818b#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

File: NOTICE.txt (hunk 1 above)
Comment (amyblais) [reply NOTICE.txt]: Chris if you can provide links for these and then I can fill in the details.

Comment (crspeller) [reply NOTICE.txt]: @amyblais Go to https://www.npmjs.com/ and search for the package names.

File: NOTICE.txt [hunk 3]
```diff
@@ -786,6 +846,10 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 ---
 
+## intl
```
Comment (crspeller) [inline NOTICE.txt]: Why is this blank?

Comment (amyblais) [inline NOTICE.txt]: I can't find it here https://www.npmjs.com/package/@types/highlight.js

Also seems that it was removed with PR https://github.com/mattermost/mattermost-webapp/commit/ca91a49d2e55c9f39ace13109a2d8ace7c015959#diff-b9cfc7f2cdf78a7f4b91a753d10865a2, but then re-added via https://github.com/mattermost/mattermost-webapp/commit/55035969cf0cade5784851a4623b34f04e1b01d8#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

Comment (crspeller) [reply NOTICE.txt]: Hmmm. Looks like a casualty of @hmhealey's merge issue.

Comment (crspeller) [reply NOTICE.txt]: You can remove it. It will be removed again soon.

**Pull request merged**